masterlist.yaml

```
@@ -852,10 +852,43 @@ plugins:
       - 'BOM_OpenMW_plazas.esp'
       - 'BOM_pathgrid_reset.esp'
       - 'multipatch.esp'
-
+   ### BCOM - Optional files ###
   - name: 'BCOM_Suran Expansion.ESP'
     url: [ 'https://www.nexusmods.com/morrowind/mods/49231/' ]
     after: [ 'BOM_pathgrid_reset.esp' ]
+  - name: 'BOM_Stavs_MGO.ESP'
+    url: [ 'https://www.nexusmods.com/morrowind/mods/49231/' ]
+    after: [ 'Beautiful cities of Morrowind.ESP' ]
+  - name: 'BOM - Taller Lighthouse.ESP'
+    url: [ 'https://www.nexusmods.com/morrowind/mods/49231/' ]
+    after: [ 'BOM_pathgrid_reset.esp' ]
+  - name: 'BOM_White_Suran.ESP'
+    url: [ 'https://www.nexusmods.com/morrowind/mods/49231/' ]
+    after:
+      - 'BCOM_Suran Expansion.esp'
+      - 'Beautiful cities of Morrowind.ESP'
+      - 'Suran_Underworld_v3.esp'
+  - name: 'SOP_Canal_02.ESP'
+    url: [ 'https://www.nexusmods.com/morrowind/mods/49231/' ]
+    after: [ 'Beautiful cities of Morrowind.ESP' ]
+  - name: 'SOP_RichPlaza_IziStyle.ESP'
+    url: [ 'https://www.nexusmods.com/morrowind/mods/49231/' ]
+    inc: [ 'Brevur of Balmora - Finally Some Good Statue Mod.ESP' ]
+    after: [ 'BOM_pathgrid_reset.esp' ]
+
+  - name: 'Brevur of Balmora - Finally Some Good Statue Mod.ESP'
+    url:
+      - 'https://www.nexusmods.com/morrowind/mods/47557/'
+      - 'https://www.nexusmods.com/morrowind/mods/49231/'
+    inc: [ 'SOP_RichPlaza_IziStyle.ESP' ]
+    after: [ 'BOM_pathgrid_reset.esp' ]
+    msg:
+      - <<: *requiresResources
+        subs: [ '[Brevur of Balmora - Finally Some Good Statue Mod](https://www.nexusmods.com/morrowind/mods/47557)' ]
+        condition: 'not file("textures/R0/Brevur.dds") and checksum("Brevur of Balmora - Finally Some Good Statue Mod.ESP", 38A76D1E)'
```

You won't need that checksum, because the file is always required.
The file just has to be present, so it doesn't matter which version you are using.
```suggestion
        condition: 'not file("textures/R0/Brevur.dds")'
```